cloud-function/src/utils.ts

```diff
@@ -59,3 +59,7 @@ const ANY_ATTACHMENT_REGEXP = createRegExpFromExtensions(
 export function isAsset(url: string) {
   return ANY_ATTACHMENT_REGEXP.test(url);
 }
+
+export function normalizeSlug(slug: string) {
+  return slug.toLowerCase().replace(/\/$/, "");
+}
```

Comment:
This is also used in `buildSitemaps.ts` and it should be called normalize path or something.

Reply:
Good point and catch, fixed in 038740d330 and 237126d022.